File: Códigos C/BEE_1214.c
#include <stdio.h>
//Média deve ser uma função

float media = 0, soma;

float mediaUp(int x[], int y) {
    int cont = 0;
    for (int j = 0; j < y; j++) {
        if (x[j] > media) {cont++;}
    }
    return (cont/(float)(y))*100;
}

int main() {
 
    int c, n;
    scanf("%d", &c);
    
    for (int i = 0; i < c; i++) {
        scanf("%d", &n);
        int temp[n];
        soma = 0;
        for (int l = 0; l < n; l++) {
            scanf("%d", &temp[l]);
            soma += temp[l];
            if (l == (n-1)) media = (float) soma/n;
        }
        printf("%.3f%%\n", mediaUp(temp, n));
    }
    
    
    return 0;
}

File: Códigos_JS/BEE_1214.js
const prompt = require('prompt-sync')();

let media = 0, soma;

function mediaUp(x) {
    let cont = 0;
    for(let j = 0; j < x.length; j++) {
        if (x[j] > media) cont++;
    }
    return ((cont/x.length)*100);
}

let c, n;
c = parseInt(prompt(''));

for (let i = 0; i< c; i++) {
    n = parseInt(prompt(''));
    let temp = [];
    soma = 0;
    for (let l = 0; l < n; l++) {
        temp[l] = parseInt(prompt(''));
        soma += temp[l];
        if (l == (n-1)) media = soma/n;
    }
    let aaa =(mediaUp(temp)).toFixed(3)
    console.log(aaa);
}